### DISCUSSION TITLE
ComboBox - FontWeight

### DISCUSSION THREAD

**Discussion by marcosgerene** [AvaloniaUI/Avalonia#13063] [Q&A] [answered]
Hi folks,

I am trying to change the ComboBox FontWeight property, but it is stucked on "semibold" or "bold".

![image](https://github.com/AvaloniaUI/Avalonia/assets/15945003/2ef01861-29e0-4fda-8ff1-5cde821a0882)

I tried even change the ComboBoxItem FontWeight but it did not work.

```
<StackPanel Classes="Padrao">
	<TextBlock Classes="Padrao" Text="My Combo" />
	<ComboBox FontWeight="Light" >
		<ComboBox.Items >
			<ComboBoxItem Content="#Value 1" FontWeight="Light" />
			<ComboBoxItem Content="#Value 2" FontWeight="ExtraBlack" />
			<ComboBoxItem Content="#Value 3" FontWeight="Normal" />
		</ComboBox.Items>
	</ComboBox>
</StackPanel>
```

I am using Avalonia v11.0.4 with FluentTheme

**Comment by timunie** [comment, marked as answer]
iirc variable fonts are not yet implemented. See also: #11092
